package.json

```diff
@@ -4,18 +4,23 @@
   "version": "0.0.0",
   "description": "Minimalistic browser-based Solidity IDE",
   "scripts": {
-    "browser-test": "nightwatch --config nightwatch.js --env local",
-    "browser-test-remote-chrome": "nightwatch --config nightwatch.js --env chrome",
-    "browser-test-remote-firefox": "nightwatch --config nightwatch.js --env default",
-    "browser-test-remote-ie": "nightwatch --config nightwatch.js --env ie",
-    "browser-test-remote-safari": "nightwatch --config nightwatch.js --env safari",
-    "browser-test-remote-parallel": "nightwatch --config nightwatch.js --env safari,ie,default,chrome",
     "build": "mkdir build; browserify src/index.js -g yo-yoify -o build/app.js -t [ babelify ]; babel --plugins=transform-es2015-template-literals build/app.js --out-file build/app.js",
+    "test": "standard; babel src --out-dir babelify-src; node test/index.js",
+    "test-browser": "npm-run-all -lpr _selenium _http-serve browser-test",
+    "test-sauce": "env-cmd .env.local node sc",
     "csslint": "csslint --ignore=order-alphabetical --errors='errors,duplicate-properties,empty-rules' --exclude-list='assets/css/font-awesome.min.css' assets/css/",
-    "downloadsolc": "rm soljson.js; wget https://ethereum.github.io/solc-bin/soljson.js",
     "lint": "standard",
-    "serve": "http-server .",
-    "test": "standard; babel src --out-dir babelify-src; node test/index.js"
+    "browser-test": "nightwatch --config nightwatch.js --env local",
+    "browser-test-remote-chrome": "env-cmd .env.local nightwatch --config nightwatch.js --env chrome",
+    "browser-test-remote-firefox": "env-cmd .env.local nightwatch --config nightwatch.js --env default",
+    "browser-test-remote-ie": "env-cmd .env.local nightwatch --config nightwatch.js --env ie",
+    "browser-test-remote-safari": "env-cmd .env.local nightwatch --config nightwatch.js --env safari",
+    "browser-test-remote-parallel": "env-cmd .env.local nightwatch --config nightwatch.js --env safari,ie,default,chrome",
+    "_http-serve": "execr --silent http-server .",
```

Comment:
for now i removed all of this and put it back to how it was, because it's now in another branch.
This branch is just for `saucelabs` related stuff now.

Otherwise - the `npm test-browser` task which is in the other branch now is supposed to run everything (thus: `npm run serve`, `npm run watch` and testing too) but supress messages from saucelabs and serve, because it spams the log and makes it hard to see the test results.

Of course, it's still possible to run the serve and watch tasks individually and then run the tests (each in it's own terminal) so you get all the logs.

Does that sound ok?